index.js

```diff
@@ -6,4 +6,9 @@
  * MIT Licensed
 */
 
-module.exports = require('./src/index').pope
+const funcs = require('./src/index');
```

Comment:
Isn't too much happening here?

Maybe we can do

```
var funcs = require('../src/index')

exports = module.exports = funcs.pope
exports.prop = funcs.prop
```

Also I will prefer `var` over `const`, since I want it to work with even legacy browsers.

Reply:
Happy to change as per above.

I did consider the compatibility issues with `const`, and so checked the existing code to see if `const` was already being used.  You might wish to change [line 54](https://github.com/poppinss/pope/blob/47484c5c73ae8f9ee3e701fe7038690c3d0763fc/src/index.js#L54) accordingly.

Reply:
Yes, it will be nice to take that off too